acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/signing/BlsSigningAcceptanceTest.java

```diff
@@ -59,7 +59,7 @@
   private static final BLSPublicKey publicKey = keyPair.getPublicKey();
 
   @ParameterizedTest
-  @EnumSource
+  @EnumSource(mode = EnumSource.Mode.EXCLUDE, names = "BLOCK_V2")
```

Comment:
Why do we need to exclude BLOCK_V2? It would good to include the block v2 in these tests for completeness.